src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -584,3 +568,14 @@ export class DocumentationItems {
     return ALL_DOCS.find(doc => doc.id === id && doc.packageName === sectionLookup);
   }
 }
+
+function processDocs(packageName: string, docs: DocItem[]): DocItem[] {
+  for (const doc of docs) {
+    doc.packageName = packageName;
+    doc.examples = exampleNames.filter(key =>
+      key.match(RegExp(`^${doc.exampleSpecs.prefix}`)) &&
+      !doc.exampleSpecs.exclude?.some(excludeName => key.indexOf(excludeName) === 0));
+  }
+
+  return docs.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
I'm not sure if it would make sense to specify `en` as a locale here as the whole page is english, but just mentioning it here in case.